d2target/d2target.go

```diff
@@ -32,11 +32,16 @@ var BorderOffset = geo.NewVector(5, 5)
 
 type Diagram struct {
 	Name        string              `json:"name"`
+	Type        string              `json:"type"`
```

Comment:
what's the intention of setting this?

Reply:
I'll remove it, was using it before to name output files.

Reply:
Type indicates whether it's a scenario, layer or step.